ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/cco-manual-workload-identity/cucushift-installer-rehearse-azure-ipi-cco-manual-workload-identity-workflow.yaml

```diff
@@ -5,6 +5,8 @@ workflow:
       - chain:  cucushift-installer-rehearse-azure-ipi-cco-manual-workload-identity-provision
     post:
       - chain:  cucushift-installer-rehearse-azure-ipi-cco-manual-workload-identity-deprovision
+    env:
+      FEATURE_SET: TechPreviewNoUpgrade
```

Comment:
@huangmingxia the workflow is changed to TP workflow, I am not sure if it need to indicate it in workflow name because I do not know the rule of workflow name.
if it does not indicate it, I think it will confuse the user.

maybe you need to contact installer team to query it.

Reply:
@kuiwang02 You are right, you need to indicate it in the workflow.

Reply:
@kuiwang02 could you help review again, much thanks.

Reply:
Just like what kuiwang mentioned, if you add the env in the workflow, need a dedicated TP workflow for it to indicate that.

For TP jobs, we encourage mates to set `FEATURE_SET` on job config level, that is why @yunjiang29 is asking if there is some way to set the ENV on cluster-bot side.